tac/guidelines/project-incubation-entry-considerations.md

```diff
@@ -26,7 +26,19 @@ A project is more likely to succeed if it publicly states its intent and estimat
 * The project should have the backing of more than one organization/individuals (i.e., the project proposers should be able to demonstrate significant, long term contribution in codebase).
 
 ### Community
-* The TAC is more likely to accept projects that have contributors familiar with open source practices. Contributing to existing LFDT projects or starting in LFDT Labs is a great place to grow this experience.
+* The TAC is more likely to accept projects that have contributors familiar with open source practices. Contributing to existing LFDT projects or starting in LFDT Labs is a great place to grow this experience. Many people contributing their code are new to running an open source project, so we want to provide some specific guidance around what is involved with managing an open source project. In your project proposal, it is expected that you will address how you will meet the following community expectations:
+
+    1. Design discussions will be moved to a public forum
```

Comment:
Mention technical discussions as well as a lot goes into those ones, and you make many important decision at the implementation level